SDPi_Supplement/asciidoc/volume2/dev-23/tf2-ch-a-mdpws-dev-23.adoc

```diff
@@ -8,23 +8,56 @@
 
 The {var_label_dev_23_message_hello} message is encoded by using {var_uri_ws_discovery_hello}[WS-Discovery Hello].
 
-===== Referenced Standards
+====== Referenced Standards
 
 * <<ws_discovery_2009>> {var_uri_ws_discovery_hello}[Section 4.1 Hello]
 * <<dpws_2009>> {var_uri_dpws_discovery}[Section 3 Discovery]
 * <<ieee_11073_20702_2016>>
 * <<ieee_11073_20701_2018>>
 
-===== Message Outline
+====== Message Outline
 
 .Hello message
 [source#vol2_clause_appendix_a_mdpws_dev_23_hello,xml]
 ----
 include::../../listings/vol2-clause-appendix-a-mdpws-dev-23-hello.xml[]
 ----
 
-===== Message Semantics
+====== Message Semantics
 
 `s12:Envelope/s12:Body/wsd:Hello/wsa:EndpointReference/wsa:Address`:: The <<actor_somds_provider>>'s <<payload_dev_23_provider_uid>> as URI.
 `s12:Envelope/s12:Body/wsd:Hello/wsd:Scopes`:: The <<actor_somds_provider>>'s <<payload_dev_23_discovery_scope>> as a list of URIs.
 
+[#vol2_clause_appendix_mdpws_dev_23_recurring_hello]
+====== Recurring Hello
+In addition to the standard trigger events of Hello messages defined in <<#vol2_clause_dev_23_message_announce_network_presence_trigger_events>>, recurring Hello messages are needed to decrease the likelihood of missed discovery messages in case of network topology changes during operation, e.g. when operational networks are extended by plugging switches (together).

```

Comment:
```suggestion
In addition to the standard trigger events of Hello messages defined in <<vol2_clause_dev_23_message_announce_network_presence_trigger_events>>, recurring Hello messages are needed to decrease the likelihood of missed discovery messages in case of network topology changes during operation, e.g. when operational networks are extended by plugging switches (together) at runtime.
```